india_compliance/gst_india/doctype/gstr_1_beta/gstr_1_beta.py

```
@@ -303,6 +303,57 @@ def get_journal_entries(month_or_quarter, year, company, filing_preference):
     return {"data": data, "posting_date": to_date}
 
 
+@frappe.whitelist()
+def get_accounts(month_or_quarter, year, company, filing_preference):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)